resources/views/hr/application/edit.blade.php

```diff
@@ -867,6 +951,20 @@ class="fa fa-close"></i>&nbsp;{{ config('constants.hr.status.rejected.title') }}
                                                     {{ $applicationRound->conducted_date->format(config('constants.display_date_format')) }}</span>
                                             @endif
                                         </div>
+                                         @if ($applicationRound->scheduled_date)
+                                                @if ($applicationRound->hangout_link)
+                                                    @if($applicationRound->meeting_duration!= NULL)
+                                                        <input type="Text" readonly="readonly" name="meeting_time"
+                                                        id="meeting_time"
+                                                        class="form-control form-control-sm"
+                                                        value="{{"$applicationRound->meeting_duration"}}">
+                                                        @else
+                                                        <button type="button" id="meet_time" value="{{$applicationRound->id}}" class="py-1 mb-0 btn btn-info btn-sm round-submit finish_interview ">Finish Interview</button>
+                                                        <input type="Text" style="display: none;" readonly="readonly" name="meeting_time"
```

Comment:
```suggestion
                                                        <input type="Text" style="display: none;" readonly="readonly" name="meetingTime"
```

Change at all the places